### DISCUSSION TITLE
PP inconsistent between osu!lazer and website

### DISCUSSION THREAD

**Discussion by Cyliann** [ppy/osu#21845] [Q&A]
I just achieved my pp record and in osu!lazer I got 107pp but the website displays 82pp. Since it's my pp record it's weighted 100%.

Which one is right?

![image](https://user-images.githubusercontent.com/33868960/209721495-9b21b6e4-bb66-4c23-878b-24d48fd617f5.png)
![image](https://user-images.githubusercontent.com/33868960/209721579-ca8b9a58-1c9b-40a4-972d-111e68555720.png)

**Comment by smoogipoo** [comment]
Can you give the score's URL?

**Reply by Theighlin** [reply]
I'm not the guy but
Score: https://lazer.ppy.sh/scores/2165483198
User: https://lazer.ppy.sh/users/16119830

**Reply by bdach** [reply]
Likely https://github.com/ppy/osu/issues/21815 again

**Reply by Cyliann** [reply]
> Can you give the score's URL?

[Here](https://lazer.ppy.sh/scores/2165483198) you go.
